components/chart/templates.md

```diff
@@ -0,0 +1,88 @@
+---
+title: No Data Template
+page_title: Chart - No Data Template
+description: The NoDataTemplate in the Chart for Blazor lets you customize the content displayed when no data is available for any series.
+slug: chart-no-data-template
+tags: telerik,blazor,chart,templates
+published: True
+position: 101
+---
+
+
+# No Data Template
+
+Starting in **version 7.0.0**, when all Chart series have no data to show, a default **No data** localizable text is displayed. You can customize this content using the `NoDataTemplate`. This is a global Chart setting and applies to all instances where a Chart series does not have any data to dispay. To change the default behavior, simply declare a `<NoDataTemplate>` tag inside a `<ChartSettings>` tag:
```

Comment:
```suggestion
Starting in **version 7.0.0**, when all Chart series have no data to show, a default layout with **No data** text is displayed over the Chart. To customize the default layout content declare a `<NoDataTemplate>` tag inside a `<ChartSettings>` tag. The content in the `<NoDataTemplate>` tag is a global Chart setting and applies to all instances.
```

Reply:
If you choose to decline the suggested change, just make sure that the instructions are not misleading in the part that the `<NoDataTemplate>` applies only to instances where a Chart series does not have any data to display. I get that from:
"This is a global Chart setting and applies to _all instances **where**_ a Chart series does not have any data to dispay."
If I understand correctly, the `<NoDataTemplate>`  applies only when **all** Chart series does not have any data.